addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -2197,13 +2197,13 @@ export class Wysiwyg extends Component {
                 const bannerElement = parseHTML(`
```

Comment:
About the commit message. Please rename "Heading 1 placeholder" into "Heading 1 Powerbox hint".

Reply:
Is this about a commit that disappeared?